app/services/courses/generate_course_name_service.rb

```diff
@@ -4,6 +4,20 @@ module Courses
   class GenerateCourseNameService
     include ServicePattern
 
+    LANGUAGES = %w[
```

Comment:
Can we use this? 

https://github.com/DFE-Digital/publish-teacher-training/blob/ce9d02f49547aed3bf15fd4b8868cddc932a61a9/app/decorators/course_decorator.rb#L8

Reply:
There are only 10 subjects in LANGUAGE_SUBJECT_CODES and 11 in LANGUAGES?

Reply:
`LANGUAGE_SUBJECT_CODES` is missing ancient Greek and ancient Hebrew. We haven't had any of those courses for years so I think its OK to leave out? Otherwise we'll have to add more logic because I think `ancient` should be lowercase and `Greek` uppercase. 

I think you're also missing `English as a second or other language` in your list. 

Either way the only thing that uses that `LANGUAGE_SUBJECT_CODES` array is to downcase non language subjects on Find, so I think the lists should be the same. See: https://github.com/DFE-Digital/publish-teacher-training/pull/3127/files

Reply:
I think we have
```
def format_subject_name(subject)
      {
        'communication and media studies' => 'Media studies',
        'english as a second or other language' => 'English',
        'physics' => course.is_engineers_teach_physics? ? ENGINEERS_TEACH_PHYSICS_TITLE : subject.to_s
      }[subject.subject_name.downcase] || subject.to_s
    end
```

Reply:
@goodviber had a chat with Simon and the two lists should be the same, so I think to keep things DRY we can stick to the old list and add the ancient subjects to it. `A1` and `A2`. 

Ignore what I said about `ancient greek` - it should be `Ancient Greek`

Reply:
and @simonwhatley list
```
- Ancient Greek
- Ancient Hebrew
- English
- French
- German
- Italian
- Japanese
- Latin
- Mandarin
- Russian
- Spanish